src/MusicStore.Spa/Helpers/AngularExtensions.cs

```diff
@@ -73,20 +73,20 @@ public static class AngularExtensions
                     ngAttributes["step"] = "any";
                 }
             }
-            else if (metadata.ModelType == typeof(DateTime))
+            else if (modelExplorer.ModelType == typeof(DateTime))
             {
-                if (string.Equals(metadata.DataTypeName, Enum.GetName(typeof(DataType), DataType.Date), StringComparison.OrdinalIgnoreCase))
+                if (string.Equals(modelExplorer.Metadata.DataTypeName, Enum.GetName(typeof(DataType), DataType.Date), StringComparison.OrdinalIgnoreCase))
                 {
                     ngAttributes["type"] = "date";
                 }
-                else if (string.Equals(metadata.DataTypeName, Enum.GetName(typeof(DataType), DataType.DateTime), StringComparison.OrdinalIgnoreCase))
+                else if (string.Equals(modelExplorer.Metadata.DataTypeName, Enum.GetName(typeof(DataType), DataType.DateTime), StringComparison.OrdinalIgnoreCase))
```

Comment:
it's a sample but choice between `modelExplorer.Metadata.ModelType` and `modelExplorer.ModelType` is important.  previous code was using the declared model type.  new code is using the runtime type.  is that correct all six places updated in this PR?